.flake8

```diff
@@ -1,15 +1,17 @@
 [flake8]
-exclude = ./build, ./docs
 docstring-convention = google
+exclude = ./build, ./docs
+extend-ignore =
+    # D105 Missing docstring in magic method
+    D105,
+    # E203 whitespace before ':' (for compliance with black)
+    E203,
+# for compliance with black
+max-line-length = 88
```

Comment:
When we adopted `black`, we decided to stick with the `flake8` 80 line limit instead

Reply:
Indeed, it's fixed!